platform/mellanox/docker-ptf-mlnx.dep

```diff
@@ -3,6 +3,7 @@
 DPATH := $($(DOCKER_PTF_MLNX)_PATH)
 DEP_FILES := $(SONIC_COMMON_FILES_LIST) $(PLATFORM_PATH)/docker-ptf-mlnx.mk $(PLATFORM_PATH)/docker-ptf-mlnx.dep
 DEP_FILES += $(SONIC_COMMON_BASE_FILES_LIST)
+DEP_FILES   += $(SONIC_COMMON_DOCKER_FILES_LIST)
```

Comment:
>   [](start = 10, length = 2)

extra blanks?

Reply:
Previously the changes are not reflecting here. 
Fixed now.